### DISCUSSION TITLE
chia coin has not arrived in my wallet

### DISCUSSION THREAD

**Discussion by KILIC77** [Chia-Network/chia-blockchain#4108] [General]
Although I won chia, it was not transferred to my wallet. Can I find out why. The block shows solved but not transferred to my wallet.

**Comment by KILIC77** [comment]
I need support.

**Comment by pluto2021** [comment]
Where to see if you won other than the GUI? Would be great if you could post the message showing that.

**Reply by KILIC77** [reply]
https://www.hizliresim.com/r9pm9z

**Comment by KILIC77** [comment]
why does not show the block I won in my wallet. Is there an error in the program? I restarted and chia is not showing in explorer either.

**Comment by mariano54** [comment]
Can you message me on keybase? sorgente711
